docs/system/system-metrics.md

```diff
@@ -916,6 +917,44 @@ This metric is [recommended][MetricRecommended].
 
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `system.disk.limit`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.system.disk.limit(metric_table) -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `system.disk.limit` | UpDownCounter | `By` | The total memory of the disk | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Is this metric equivalent to adding up another `system.disk` metric across a particular dimension? (I don't think so, but just double checking)

Reply:
I don't think so. All the current `system.disk` metrics are related to the performance of the disk (read/write io and read/write operationd), not the actual usage.